src/SConscript

```diff
@@ -58,11 +61,9 @@ def scons():
             env.Install(os.path.join('$PREFIX', 'include', 'daos_srv'),
                         'include/daos_srv/%s' % header)
     for header in HEADERS_GURT:
-        env.Install(os.path.join('$PREFIX', 'include', 'gurt'),
-                    'include/gurt/%s' % header)
+        env.Install(os.path.join('$PREFIX', 'include', 'gurt'), 'include/gurt/%s' % header)
```

Comment:
(pylint-consider-using-f-string) Formatting a regular string which could be a f-string